api_test.go

```diff
@@ -139,17 +138,4 @@ func TestNetAPI(t *testing.T) {
 	if err == nil {
 		t.Errorf("unexpected HashLookup success, wanted HTTP request canceled")
 	}
-
-	// Test for detection of incorrect protobufs.
```

Comment:
Is there a particular reason we are deleting this test?

Reply:
Yes. The updated vendored protobuf code doesn't produce this error anymore. It seems like unknown fields are now stored with the target object when unmarshalling, probably to avoid breakage when changing a protocol.